.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
It's also possible that using setup.cfg will remove the need to specify those configurations again in the Github Actions YAML? Worth trying. If not we'll need to double check that those commands exactly match what's in config, because we don't want different checks running locally vs. in CI.

Reply:
You were correct that setup.cfg worked for CI so I simplified the commands